lib/scrape.js

```diff
@@ -369,14 +377,19 @@ async function fetchDates() {
 }
 
 ;(async () => {
-  const data = await fetchOrgsWithData()
+  const orgs = await fetchOrgsWithData()
   const dates = await fetchDates()
 
   // sort data by completed_task_instance_count
-  data.sort(
+  orgs.sort(
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
 
+  const data = {
+    orgs: sortKeys({ orgs }, { deep: true }).orgs,
+    lastUpdated: new Date(),
```

Comment:
This isn't used later, right?

rss.js script is reading from feed_items.json

But this is bad in general, we want the json structure to be stable from time to time, so that we can easily process it later and do some analytics in the end.

Reply:
This is used [here](https://github.com/coala/gci-leaders/pull/95/files#diff-9676cbcb5938ab810f7fa82a528abc4eR46) to ensure that we're not repeating the same data changes in the RSS. What's the best alternative you can think of?

Reply:
Maybe create a new file that holds the last updated time.

We have many generated files (org data, blog planet, dates) and that'll give the last updated time to all of that data.